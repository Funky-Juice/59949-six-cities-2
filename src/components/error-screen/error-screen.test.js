import renderer from 'react-test-renderer';
import ErrorScreen from './error-screen';

const mockMsg = `Test error message`;

it(`ErrorScreen correctly renders`, () => {
  const tree = renderer
    .create(<ErrorScreen
      errorMessage={mockMsg}
    />).toJSON();

  expect(tree).toMatchSnapshot();
});
