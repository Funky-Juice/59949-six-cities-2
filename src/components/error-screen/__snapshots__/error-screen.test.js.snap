// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorScreen correctly renders 1`] = `
<div
  className="page page--main"
>
  <main
    className="page__main page__main--index page__main--index-empty"
  >
    <div
      className="cities"
    >
      <div
        className="cities__places-container cities__places-container--empty container"
      >
        <section
          className="cities__no-places"
        >
          <div
            className="cities__status-wrapper cities__status-wrapper--error tabs__content"
          >
            <b
              className="cities__status"
            >
              Test error message
            </b>
          </div>
        </section>
        <div
          className="cities__right-section"
        />
      </div>
    </div>
  </main>
</div>
`;
